db/seed.js

```diff
@@ -20,5 +20,7 @@ for (let i = 0; i < 99; i += 1) {
 }
 
 for (let i = 0; i < 98; i += 1) {
-  db.addReservation(generateRestaurantId(), generateDate(), generateTime(), generatePartySize());
+  db.addReservation(generateRestaurantId(), generateDate(), generateTime(), generatePartySize(), () => {
```

Comment:
Consider adding a console.log in your success callback to indicate a successful reservation entry, or add a comment describing the reason for the empty callback